pep-0001.txt

```diff
@@ -293,14 +294,16 @@ the time when when Python was led by `a BDFL <Python's BDFL_>`_.
 Any legacy references to "BDFL-Delegate" should be treated as equivalent to
 "PEP-Delegate".
 
-Individuals offering to serve as PEP-Delegate should notify the Steering
-Council, PEP authors and PEP sponsor of their intent to self-nominate.
+Individuals offering to serve as PEP-Delegate should notify the PEP authors
+and PEP sponsor of their intent to self-nominate, and submit their request as
```

Comment:
I'm getting to a point of cognative dissonance with "PEP" in this sentence! Perhaps:
```suggestion
An individual offering to serve as PEP-Delegate must notify the relevant 
authors and sponsor (where a sponsor exists) for the PEP. The request must be 
formally submitted by opening
```

Reply:
This could be combined with @brettcannon's suggestion that I just saw.

Reply:
Yeah; I'll have to do so manually

Reply:
Done